tests/manifests/default/kustomization.yaml

```diff
@@ -3,6 +3,8 @@ kind: Kustomization
 
 resources:
   - ../../../config/default
+  - ./controller_def_rc0.0.18.yaml
+  - ./controller_def_rc0.0.19.yaml
```

Comment:
it's easier to create these controller definitions that are single files rather than have one big long file of all of the definitions

I think in the future it might be nice if we can just copy the entire manifest from the open source file and not even worry about needing to remove anything by hand. (I noticed that when I did this, the `Service` Type wasn't an [includedResource](https://github.com/numaproj/numaplane/blob/main/config/install.yaml#L1402) so that failed, but maybe we can change that?)